src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -317,6 +317,7 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
     const isValidKey =
       Events.isNumberKey(event) ||
       Events.isTabKey(event) ||
+      Events.isEnterKey(event) ||
```

Comment:
matches change added in #6837

Reply:
yep it will be rebased this is just so I can add the tests in as it wasn't merged when this was raised

Reply:
Rebased now fyi